src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -75,6 +75,7 @@ public function __construct(
     /**
      * @Transform :shippingMethodName shipping method
      * @Transform shipping method :shippingMethodName
+     * @Transform /^"([^"]*)" shipping$/
```

Comment:
Yeah, as our dictionary of new steps grows, we should be more specific here. Otherwise we can suffer debugging crazy transformations.